modin/pandas/test/test_general.py

```diff
@@ -725,6 +737,24 @@ def test_to_numeric(data, errors, downcast):
     df_equals(modin_result, pandas_result)
 
 
+@pytest.mark.parametrize("retbins", bool_arg_values, ids=bool_arg_keys)
+def test_qcut(retbins):
+    # test case from https://github.com/modin-project/modin/issues/5610
+    pandas_series = pandas.Series(range(10))
+    modin_series = pd.Series(range(10))
+    pandas_result = pandas.qcut(pandas_series, 4, retbins=retbins)
+    modin_result = pd.qcut(modin_series, 4, retbins=retbins)
+    if retbins:
+        df_equals(modin_result[0], pandas_result[0])
+        assert_index_equal(
+            modin_result[0].cat.categories, pandas_result[0].cat.categories
```

Comment:
We can use `df_equals` function for comparison, since we already do this in a test named `test_cat_categories`.

Reply:
fixed